govspeak.gemspec

```diff
@@ -31,7 +31,7 @@ library for use in the UK Government Single Domain project'
 
   s.add_dependency "actionview", ">= 6"
   s.add_dependency "addressable", ">= 2.3.8", "< 3"
-  s.add_dependency "govuk_publishing_components", ">= 35.1"
+  s.add_dependency "govuk_publishing_components", ">= 35.1", "< 38"
```

Comment:
Given `v38` has been about since April, do you know why the tests are failing only now, not on other recent PRs?

Reply:
Looks like it was 38.4.2 that introduced the extra attributes (released two weeks ago), so I'll pin specifically to the version before that.

Reply:
optional: even better if you can link to a bug (e.g. a GitHub issue) so folks can tell when the workaround might no longer be needed

Reply:
We've now got a PR to bump govuk_publishing_components, where we can focus on fixing the failing test: https://github.com/alphagov/govspeak/pull/323.